modules/user-guide/pages/setting-up-the-gitlab-authorized-application.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: PROCEDURE
+:description: Setting up the GitLab-authorized application
+:keywords: configuring-authorization, configure-authorization, private-repository, private-git-repository, private-repo, private-git-repo, private-gitlab, private, gitlab, gitlab-repo, gitlab-repository
+:navtitle: Setting up the GitLab-authorized application
+// :page-aliases:
+
+[id="setting-up-the-gitlab-authorized-application_{context}"]
+= Setting up the GitLab-authorized application
+
+To be able to access a private Git repository that is hosted using GitLab, set up a GitLab-authorized application as follows:
+
+.Prerequisites
+
+* You are logged in to GitLab.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. Click your avatar *>* *Edit profile* *>* *Applications*.
+//Applicable only to gitlab.com: Go to link:https://gitlab.com/-/profile/applications[]. max-cx
+
+. Enter *{prod-short}* as the *Name*.
+
+. Enter `{prod-url}/api/oauth/callback` as the *Redirect URI*.
+
+. Check the *Confidential* and *Expire access tokens* checkboxes.
+
+. Under *Scopes*, check the `api`, `write_repository`, and `openid` checkboxes.
+
+. Click *Save application*.
+
+. Save and then encode to Base64 the *GitLab Application ID* like so:
++
+[source,subs="+quotes,+attributes,+macros"]
+----
+$ echo -n '__<gitlab_application_id>__' | base64
```

Comment:
I'm not sure why the italics is rendered correctly here 
<img width="810" alt="image" src="https://user-images.githubusercontent.com/83611742/160721835-6d4d27d9-9795-4c2e-972c-526e2a5d82b9.png">


but not here: https://github.com/eclipse-che/che-docs/pull/2242/files#r837953664

Maybe it is the `[source,subs="+quotes,+attributes,+macros"]`?

Reply:
Correct. My bad. Fixing it.